scripts/check-release.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+
+if [[ "$#" -lt "2" || "$#" -gt 3 ]]
+then
+	echo "error: incorrect number of arguments" >&2
+	echo "Usage: $0 <DIST_URL> <MAVEN_URL> [LOCAL_RELEASE_DIR]" >&2
+	exit 1
+fi
+
+# URL of release candidate directory in dev/dist/, e.g. https://dist.apache.org/repos/dist/dev/daffodil/1.0.0-rc1
+DIST_URL=$1
+
+# URL of maven staging repository, e.g. https://repository.apache.org/content/repositories/orgapachedaffodil-1234
+MAVEN_URL=$2
+
+# optional path to release directory built by running the daffodil-build-release
+# container. If not provided, only signature/checksum checks are done
+LOCAL_RELEASE_DIR=$3
+
+require_command() {
+	command -v "$1" &> /dev/null || { echo "error: command $1 not found in PATH"; exit 1; }
+}
+
+# error early if needed tools are missing
+require_command diff
+require_command gpg
+require_command md5sum
+require_command sha1sum
+require_command sha512sum
+require_command wget
```

Comment:
Missing `require_command rpm` since this uses rpm to verify the embedded signature